lib/lit/cache.rb

```diff
@@ -303,7 +308,11 @@ def split_key(key)
     end
 
     def find_or_create_localization_key(key_without_locale)
-      localization_key = Lit::LocalizationKey.where(localization_key: key_without_locale).first_or_create!
+      localization_key = Lit::LocalizationKey.find_or_initialize_by(
+        localization_key: key_without_locale
+      )
+      localization_key.is_deleted = false
```

Comment:
what will happen ie. with scope fallback chains? Ie. simple_form gem calls lots of keys as fallback, won't adding new form input reenable previously deleted (hidden) keys?

Reply:
I'm not sure if I get it right. If given key is not present in any view anymore it's still possible that another key will fallback to it?